DTS-STN/Alpha-Site
# Analytics implementation

# Description

[Go through the Adobe Analytics Implementation Testing Process manual alongside the Implementation Run Testing Checklists](https://trello.com/c/Ov9CbyGo/287-go-through-the-adobe-analytics-implementation-testing-process-manual-alongside-the-implementation-run-testing-checklists)

I added a meta tag (<meta name="dcterms.issued" content="yyyy-mm-dd"/). The tag is for when the page was originally published on the site for analytics.

## Acceptance Criteria

Every page should have the meta tage with the name dcterms.issued

## Test Instructions

1. Run or just check the code for each page
2. Make sure they have a date.issued meta tag

## Checklist


## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)
